packages/node_modules/@webex/internal-plugin-mercury/test/unit/spec/mercury.js

```diff
@@ -400,6 +400,65 @@ describe('plugin-mercury', () => {
             });
         });
       });
+
+      describe('Websocket proxy agent', () => {
+        afterEach(() => {
+          delete webex.config.defaultMercuryOptions;
+        });
+
+        it('connects to Mercury using http proxy agent', () => {
+          const testProxyUrl = 'http://proxyurl.com:80';
+
+          webex.config.defaultMercuryOptions = {agent: {proxy: {href: testProxyUrl}}};
+          const promise = mercury.connect();
+
+          assert.isFalse(mercury.connected, 'Mercury is not connected');
+          assert.isTrue(mercury.connecting, 'Mercury is connecting');
+          mockWebSocket.open();
+
+          return promise
+            .then(() => {
+              assert.isTrue(mercury.connected, 'Mercury is connected');
+              assert.isFalse(mercury.connecting, 'Mercury is not connecting');
+              assert.calledWith(socketOpenStub, sinon.match(/ws:\/\/example.com/), sinon.match.has('agent',
+                sinon.match.has('proxy', sinon.match.has('href', testProxyUrl))));
+            });
+        });
+
+        it('connects to Mercury using https proxy agent', () => {
```

Comment:
This is a duplicate test.

Reply:
I guess you're right, without the HTTP_PROXY/HTTPS_PROXY env vars, there's no need for two different tests here